src/libraries/System.IO.Ports/tests/SerialPort/Encoding.cs

```diff
@@ -111,7 +111,9 @@ public void Encoding_ISCIIAssemese()
         public void Encoding_UTF7()
         {
             Debug.WriteLine("Verifying UTF7Encoding Encoding");
+#pragma warning disable BCL0001 // Encoding.UTF7 property is obsolete
```

Comment:
Do obsoletion IDs need to be numerical?  If not, have we considered using more descriptive names that would avoid the need for a comment?  May not be a great idea to break from convention even if it's possible.